core/src/test/scala/integration/kafka/api/ProducerCompressionTest.scala

```diff
@@ -103,12 +123,37 @@ class ProducerCompressionTest extends QuorumTestHarness {
       // make sure the fetched message count match
       consumer.assign(Collections.singleton(tp))
       consumer.seek(tp, 0)
-      val records = TestUtils.consumeRecords(consumer, numRecords)
+      val records = TestUtils.consumeRecords(consumer, numRecords*3)
```

Comment:
nit

<space>*<space>3

Reply:
Thanks Divij! I'll merge this first to unblock v3.6.0 release.